Library/Homebrew/requirements/gpg_requirement.rb

```diff
@@ -2,7 +2,7 @@
 
 class GPGRequirement < Requirement
   fatal true
-  default_formula "gpg"
+  default_formula "gnupg2"
```

Comment:
Long-term support, essentially. 1.x only receives important bug fixes, upstream doesn't really recommend it for much beside portability and everyone else is moving away from it. I'd rather move away on our timetable so we can control the transition.

There's enough hacks in other projects usage in terms of _"If we're using 2.x use this feature, if we're using 1.x don't bother"_ that I think by defaulting to `gnupg` 1.x we're contributing to that.